src/main/kotlin/org/jmailen/gradle/kotlinter/tasks/FormatTask.kt

```diff
@@ -13,6 +14,7 @@ import java.io.File
 
 open class FormatTask : SourceTask() {
 
+    @Internal
```

Comment:
We do this to prevent incremental build, which doesn't work quite right if you revert after since this task modifies input files.